.rubocop_todo.yml

```diff
@@ -331,7 +331,17 @@ Style/SymbolProc:
 # Cop supports --auto-correct.
 # Configuration parameters: EnforcedStyleForMultiline, SupportedStylesForMultiline.
 # SupportedStylesForMultiline: comma, consistent_comma, no_comma
-Style/TrailingCommaInLiteral:
+Style/TrailingCommaArrayInLiteral:
+  Exclude:
```

Comment:
Instead of excluding these files, let's go ahead and fix them in this PR. `rubocop` might even be able to do it automatically.

Reply:
I assumed removing them from the .robocop_todo.yml was the right way to not exclude them. I manually added commas where rubocop complained.